### ISSUE TITLE
feat: search bar for revanced settings 

### ISSUE THREAD

**Issue by triviahackfanhh** [inotia00/ReVanced_Extended#2001]
### Application

YouTube Music

### Feature description

feat:  search bar (search options) for revanced settings. Hey, can't you provide a setting search bar, to quick jump to the desired revanced settings. For normal users like us, its really difficult to easily jump to the desired revanced settings. We need to check inside all the options to find specific settings. I think it's not a big task for you. You can easily Provide... Thanks you 

### Motivation

feat:  search bar (search options) for revanced settings. Hey, can't you provide a setting search bar, to quick jump to the desired revanced settings. For normal users like us, its really difficult to easily jump to the desired revanced settings. We need to check inside all the options to find specific settings. I think it's not a big task for you. You can easily Provide... Thanks you 

### Acknowledgements

- [X] This is not a feature request for YouTube.
- [X] This issue is not a duplicate of an existing feature request.
- [X] I have chosen an appropriate title.
- [X] All requested information has been provided properly.
- [X] I have written the title and contents in English.

**Comment by Vipin2003** [comment]
+1 for this...Searching a specific patch in different categories is very hard

**Comment by 15sky** [comment]
Yes please, YouTube has this and adding it to YT Music would help a lot and making finding the setting you want so much easier
